tests/acceptance/features/webUIFilesActionMenu/versions.feature

```diff
@@ -83,3 +83,13 @@ Feature: Versions of a file
     When the user re-logs in as "Alice" using the webUI
     And the user browses to display the "versions" details of file "lorem-file.txt"
     Then the versions list should contain 0 entries
+
```

Comment:
```suggestion

  @issue-5534
```

Actually #5534 is a closed PR. This needs to point to an issue.

Reply:
It's not issue. It's PR. Maybe something like @web-pull-5548?

Reply:
The PR will be in closed/merged state when it is approved and merged. So developers will not notice it when scheduling work in sprints. We should raise an issue that describes the problem, and link to the issue number here and in the expected-failures files.

Reply:
A mistake on the backend side. I created issue in ocis and added points to an issue